src/tcgwars/logic/impl/gen8/LegendaryHeartbeat.groovy

```diff
@@ -1056,8 +1056,10 @@ public enum LegendaryHeartbeat implements LogicCardInfo {
           text " Both Active Pokémon are Knocked Out."
           energyCost M, M, C, C
           onAttack {
-            new Knockout(defending).run(bg)
-            new Knockout(self).run(bg)
+            afterDamage {
+              new Knockout(defending).run(bg)
+              new Knockout(self).run(bg)
+            }
```

Comment:
I am curious what does this change?

Reply:
This was primarily an experiment that turned out to not have any effect. Committing it was an accident, but it shouldn't affect anything IIRC.